GUI/python source/UMSATSGUI_support.py

```diff
@@ -174,19 +186,21 @@ def destroy_window():
     top_level.destroy()
     top_level = None
 
-def command1_enter(e,display):
-    #print('command1_enter\n')
-    writeCommand(e.widget,display,1,cmd1Text.get())
+
+def command1_enter(e, display):
```

Comment:
These function names should be more descriptive name than "command1" and "command2"?